<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="createDatabase">
       <property name="text">
        <string>Create Database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeDuplicates">
       <property name="text">
        <string>Remove Duplicates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="queryStock">
       <property name="text">
        <string>Query Stock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="authenticate">
       <property name="text">
        <string>Authenticate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fetchStock">
       <property name="text">
        <string>Fetch Stock Price</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateAll">
       <property name="text">
        <string>Update All Prices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tickerInput">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="dataOutput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>841</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Console Output</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
